CMakeLists.txt

```diff
@@ -191,8 +191,15 @@ rocm_create_package(
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/lib)
 set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/lib)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/bin)
-
 add_subdirectory(src)
 add_subdirectory(doc)
 add_subdirectory(test)
 add_subdirectory(tools)
+
+set(DEST_DIR ${CMAKE_BINARY_DIR})
+file (COPY ${CMAKE_SOURCE_DIR}/src/py/backend/backend.py DESTINATION ${DEST_DIR}/lib/onnx_migraphx/.)
+file (COPY ${CMAKE_SOURCE_DIR}/src/py/backend/backend_rep.py DESTINATION ${DEST_DIR}/lib/onnx_migraphx/.)
+file (COPY ${CMAKE_SOURCE_DIR}/src/py/backend/__init__.py DESTINATION ${DEST_DIR}/lib/onnx_migraphx/.)
+file (COPY ${CMAKE_SOURCE_DIR}/test/py/onnx_backend_test_series.py DESTINATION ${DEST_DIR}/.)
+file (COPY ${CMAKE_SOURCE_DIR}/test/py/testdata DESTINATION ${DEST_DIR}/.)
+
```

Comment:
How are these python scripts invoked for the tests?

Reply:
I moved the file onnx_backend_test.py to the folder test/py, and added one line to the CMakeList.txt in this folder as:
add_py_test(backend onnx_backend_test WORKING_DIRECTORY ${TEST_ONNX_DIR})
now it can be invoked.